velox/exec/Task.cpp

```diff
@@ -1507,6 +1507,41 @@ ContinueFuture Task::requestPauseLocked(bool pause) {
   return makeFinishFutureLocked("Task::requestPause");
 }
 
+// Checks if it is likely that the reservation on 'tracker' can be
```

Comment:
This comment duplicates the comment in the header file. Also, "This will be moved to Task..." is no longer needed.